jekyll/_cci2/ip-ranges.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: classic-docs
+title: "IP ranges"
+short-title: "IP ranges"
+description: "Enable CircleCI jobs to go through a set of well-defined IP address ranges"
+categories: []
+order: 70
+version:
+ - Cloud
+---
+
+
+Enable CircleCI jobs to go through a set of well-defined IP address ranges.
+
+
+* TOC
+{:toc}
+
+## Overview
+{: #overview }
+
+IP ranges is a feature for CircleCI customers who need to configure IP-based access to their restricted environments. As part of this feature, CircleCI provides a list of well-defined IP address ranges associated with the CircleCI service. CircleCI jobs that have this feature enabled will have their traffic routed through one of the defined IP address ranges.
+
+The feature is currently available in preview to customers on any [paid pricing plan](https://circleci.com/pricing/).
+
+## IP ranges: use cases
+{: #usecases }
+
+	IP ranges empowers customers to limit inbound connections to their infrastructure to IP address ranges that are verifiably associated with CircleCI.  
+
+Some example customer use cases where IP-based restricted access might be desired include:
+- Accessing private artifact repositories 
+- Pulling dependencies from a CocoaPods proxy hosted behind a firewall
+- Running test cases on an internal environment 
+- Performing integration testing against private AWS resources
+- Deploying an internal app with sensitive data
+- Granting access to a production network 
+
+Previous to offering IP ranges, the only solution CircleCI offered customers in need of static IP addresses which could be configured and controlled was [CircleCI’s Runner](https://circleci.com/docs/2.0/runner-overview/). IP ranges now enables customers to meet their IP-based security and compliance requirements using their existing workflows and platform. 
```

Comment:
We try to talk directly to the developer in docs rather than talk about 'customers' but I defer to @thomasr8 here 🙏  (connected to another suggestion above to take out general customer language).